fourkitchens/suhumsci
# Shs 4863 labels non devs

## Summary
This hides text format changes and text format help text from non-developer roles. 

## Steps to Test
1. Under a developer account, go to a flexible page and test every component for the following: when there exists a WYSIWYG text area, nothing has changed (text format dropdown and text help link still exist). Screenshot below.
2. Under a non-developer account, do the same as above except the text format dropdown and text help link should not display. Screenshot below.
3. Do steps 1 and 2 for every content type (luckily there aren't components just text areas for these).

## Screenshots
Dev:
![Screenshot 2023-04-26 at 11 16 52 AM](https://user-images.githubusercontent.com/8405274/234623150-5d2aa37e-2ebb-4204-9f60-2d7002284698.png)

Non-dev:
![Screenshot 2023-04-26 at 11 17 11 AM](https://user-images.githubusercontent.com/8405274/234623196-20092413-7a1e-4955-b7c3-9ad52fe5df6f.png)

## Notes
- I branched off a pre-existing PR because it had the CSS file set up already.
- The ticket went back and forth between a backend solution and a CSS solution. I chose CSS because it's a couple lines of code and already an established pattern based on a previous PR.
